posthog/queries/breakdown_props.py

```diff
@@ -147,6 +147,8 @@ def get_breakdown_prop_values(
         cast_as_float=filter.using_histogram,
     )
 
+    sample_clause = f"SAMPLE {filter.sampling_factor}" if filter.sampling_factor else ""
```

Comment:
Interpolation is kind of :( as a practice but checked the code and it seems defensible here.

Reply:
yeah I might work on a refactor once everything is in to do params everywhere